problems/[230612] Implement Queue using Stacks.js

```diff
@@ -0,0 +1,52 @@
+const MyQueue = function () {
+  this.stack1 = [];
+  this.stack2 = [];
+};
+
+/**
+ * @param {number} x
+ * @return {void}
+ */
+MyQueue.prototype.push = function (x) {
+  this.stack1.push(x);
+};
+
+/**
+ * @return {number}
+ */
+MyQueue.prototype.pop = function () {
+  while (this.stack1.length !== 0) {
+    this.stack2.push(this.stack1.pop());
+  }
+
+  const result = this.stack2.pop();
+
+  while (this.stack2.length !== 0) {
+    this.stack1.push(this.stack2.pop());
+  }
+
+  return result;
+};
+
+/**
+ * @return {number}
+ */
+MyQueue.prototype.peek = function () {
+  return this.stack1.length === 0 ? null : this.stack1[0];
```

Comment:
```suggestion
  return this.stack1.length && this.stack1[0];
```
위와 같이 수정할 수 있을 것 같아요.